strings/base_com_ptr.h

```diff
@@ -7,16 +7,27 @@ WINRT_EXPORT namespace winrt
 
 namespace winrt::impl
 {
+    struct capture_decay
+    {
+        void** result;
+
+        template <typename T>
+        operator T** ()
```

Comment:
Maybe check if `T` derives from IUnknown?

Reply:
In most cases `T` will be `void` - the whole point here is to throw away type safety and just take whatever is provided, as long as it's a `T**`.